libs/api-client/src/Network/Wire/Client/API/Conversation.hs

```diff
@@ -35,6 +35,8 @@ postOtrMessage cnv msg = sessionRequest req rsc readBody
         $ empty
     rsc = status201 :| [status412]
 
+-- | Add a user and (in case of success) return the event corresponding to
```

Comment:
It's a list of users.  What event is returned in case some additions are successful, but not all?